timesketch/api/v1/resources_test.py

```diff
@@ -486,6 +483,166 @@ def test_get_sigma_rule_list(self):
+            "logsource": {"product": "linux", "service": "shell"},
+            "title": "Suspicious Installation of bbbbbb",
+        }
+    }
+
+    @mock.patch(
+        "timesketch.api.v1.resources.OpenSearchDataStore", MockDataStore
+    )
+    def test_post_sigma_resource(self):
+        """Authenticated request to POST an sigma rule."""
+        MOCK_SIGMA_RULE = """
+title: Suspicious Installation of bbbbbb
+id: 5266a592-b793-11ea-b3de-bbbbbb
+description: Detects suspicious installation of bbbbbb
+references:
+    - https://rmusser.net/docs/ATT&CK-Stuff/ATT&CK/Discovery.html
+author: Alexander Jaeger
+date: 2020/06/26
+modified: 2022/06/12
+logsource:
+    product: linux
+    service: shell
+detection:
+    keywords:
+        # Generic suspicious commands
+        - '*apt-get install bbbbbb*'
+    condition: keywords
+falsepositives:
+    - Unknown
+level: high
+"""
+
+        self.login()
+
+        sigma = dict(
+            rule_uuid="5266a592-b793-11ea-b3de-bbbbbb",
+            title='Suspicious Installation of bbbbbb',
+            description='Detects suspicious installation of bbbbbb',
+            rule_yaml=MOCK_SIGMA_RULE,
+        )
+
+        # Create a first rule
+        response = self.client.post(
+            "/api/v1/sigmarule/5266a592-b793-11ea-b3de-bbbbbb/",
+            data=json.dumps(sigma),
+            content_type="application/json",
+        )
+        self.assertIn('bbbbbb', response.json['objects'][0]["rule_uuid"])
+        self.assertIn(
+            'bbbbbb',
+            response.json['objects'][0]["description"],
+        )
+        self.assertIn(
+            'shell',
+            response.json['objects'][0]["rule_yaml"],
+        )
+        self.assertEqual(response.status_code, HTTP_STATUS_CODE_CREATED)
+        # Now GET the ressources
+        response = self.client.get(
+            "/api/v1/sigmarule/5266a592-b793-11ea-b3de-bbbbbb/"
+        )
+
+        self.assertIsNotNone(response)
+        self.assertEqual(response.status_code, HTTP_STATUS_CODE_OK)
+        self.assertIn('bbbbbb', response.json['objects'][0]["rule_uuid"])
+        self.assertIn(
+            'bbbbbb',
+            response.json['objects'][0]["description"],
+        )
+        self.assertIn(
+            'shell',
+            response.json['objects'][0]["rule_yaml"],
+        )
+
+        # Attempt to add the same thing again with different content
+        new_rule = """
+title: Suspicious Installation of cccccc
+id: 5266a592-b793-11ea-b3de-cccccc
+description: Detects suspicious installation of cccccc
+references:
+    - https://rmusser.net/docs/ATT&CK-Stuff/ATT&CK/Discovery.html
+author: Alexander Jaeger
+date: 2020/06/26
+modified: 2022/06/12
+logsource:
+    product: linux
+    service: shell
+detection:
+    keywords:
+        # Generic suspicious commands
+        - '*apt-get install cccccc*'
+    condition: keywords
+falsepositives:
+    - Unknown
+level: high
+"""
+
+        response = self.client.post(
```

Comment:
No need to store the response to this `post` call if you're not going to do anything on it.